archive/terraform/api_gw_lambda/api_gateway.tf

```diff
@@ -50,7 +49,7 @@ resource "aws_api_gateway_deployment" "test" {
     "aws_api_gateway_integration.lambda_root",
   ]
 
-  rest_api_id = "${aws_api_gateway_rest_api.archive_asset_lookup.id}"
+  rest_api_id = "${aws_api_gateway_rest_api.api.id}"
   stage_name  = "prod"
 
   # keep this bit around as might require it for terraform to update the deployment
```

Comment:
A note about this - without the commented code, any changes to the API gateway setup won't get deployed, so they'd have to be manually actioned in the UI or via script. The side-effect of the commented code is that every time the Terraform is applied then a deployment to API gateway will occur. This was more of an issue when I was developing a basic HTTP auth custom authorizer and had to manually deploy it after each change.

Reply:
Eep. That's not great, but doesn't seem like a regression.